docs/reference/README.md

```diff
@@ -301,6 +302,14 @@ The wallets below support FIL tokens. We recommend verifying that these have bee
 - A Filecoin light wallet example is in the [Filecoin.js library](https://github.com/filecoin-shipyard/filecoin.js)
 - [Filecoin Rosetta API Proxy](https://github.com/Zondax/rosetta-filecoin) - [Rosetta](https://www.rosetta-api.org/) is an API standard created by Coinbase for a consistent interface to many chains for wallets and exchanges
 
+### Wallet Addresses
```

Comment:
For clarity in this section, it might be good to add bold sub headers for "Public Key Addresses" and "ID Addresses", so the two classes are visually called out.

Can you also add a sentence about why ID addresses are created and what they're used for? Which to use when? Can I use my ID address when transferring to Coinbase or do I need to use my PK address? Is not clear.